crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -531,6 +531,11 @@ impl StandardMaterial {
     /// Flip the texture coordinates of the material.
     pub fn flip(&mut self, horizontal: bool, vertical: bool) {
         if horizontal {
+            // self.uv_transform *= Self::FLIP_HORIZONTAL will not work as expected
+            // because it would be equivalent of
+            // self.uv_transform = self.uv_transform * Self::FLIP_HORIZONTAL;
+            // which means first flip, then original uv_transform application,
+            // which is reverse to expected sequence
```

Comment:
Small, non-blocking, nit. If you prefer your wording, feel free to keep it. :)

```suggestion
            // Multiplication of `Affine2` is order dependent, which is why
            // we do not use the `*=` operator.
```